docs/references/web-ui-environment-variables.md

```diff
@@ -0,0 +1,229 @@
+---
+id: web-ui-environment-variables
+title: Temporal Web UI environmental variables
+description: How to set  environmental variables for Temporal Web UI.
+sidebar_label: Web UI env var
+tags:
+  - docker
+  - webui
+  - ui server
+  - reference
+---
+
+<!-- This file is generated. Do not edit it directly. -->
+
+You can use environment variables to dynamically alter the configuration of your Temporal Web UI.
+
+These can be used in many environments such as Docker.
+For example:
+
+<!-- ```
+docker run \
+    -e TEMPORAL_ADDRESS=127.0.0.1:7233 \
+    -e TEMPORAL_UI_PORT=8080 \
+    -e TEMPORAL_AUTH_ENABLED=true \
+    -e TEMPORAL_AUTH_PROVIDER_URL=https://accounts.google.com \
+    -e TEMPORAL_AUTH_CLIENT_ID=xxxxx-xxxx.apps.googleusercontent.com \
+    -e TEMPORAL_AUTH_CLIENT_SECRET=xxxxxxxxxxxxxxx \
+    -e TEMPORAL_AUTH_CALLBACK_URL=https://xxxx.com:8080/auth/sso/callback \
+    -e TEMPORAL_UI_ENABLED=true \
+    -e TEMPORAL_OPENAPI_ENABLED=true \
+    -e TEMPORAL_TLS_CA=../ca.cert \
+    -e TEMPORAL_TLS_CERT=../cluster.pem \
+    -e TEMPORAL_TLS_KEY=../cluster.key \
+    -e TEMPORAL_TLS_ENABLE_HOST_VERIFICATION=true \
+    -e TEMPORAL_TLS_SERVER_NAME=tls-server \
+    temporalio/ui:<tag>
+``` -->
+
+The environment variables are defined in the [UI server configuration template file](https://github.com/temporalio/ui-server/blob/main/docker/config_template.yaml) and described in more detail below.
+
+## `TEMPORAL_ADDRESS`
+
+The [Frontend Service](/clusters#frontend-service) address for the Temporal Cluster.
+This environmental variable can be set [in the base configuration file](/references/web-ui-configuration#temporalgrpcaddress) using `temporalGrpcAddress`.
+
+This variable is required for setting other environmental variables.
+`TEMPORAL_ADDRESS` is also needed when setting up [Cross-Origin Resource Sharing](/references/web-ui-configuration#cors).
+
+## `TEMPORAL_UI_PORT`
+
+The port used by the Temporal WebUI Server and the HTTP API.
+
+Defining this variable allows the Cluster and API to communicate effectively.
+
+This variable is needed for `TEMPORAL_OPENAPI_ENABLED` and all auth-related settings to work properly.
+`TEMPORAL_UI_ENABLED` requires a valid port number.
+
+## `TEMPORAL_PUBLIC_PATH`
+
+The location in the Web UI Server where everything is stored.
```

Comment:
could we update this line as well

updated the text here https://github.com/temporalio/documentation/pull/1640/files#diff-e35af383f2c268ac1b28e8c3bc699e497eaf3d473683478c0edcac6d732ca52cR54 just needs copying